utils/thapilog_set_ts.rb

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env ruby
+
+# Fix timestamps in a babeltrace_thapi -c log file
```

Comment:
Should we just fix babeltrace_thapi?
What does it do?

Reply:
No bug in babeltrace_thapi, it is just when hand editing log files, timestemps get messed up and manually setting them can be tedious. I can change the comment so is more clear, like "Set timestemps in a manually edited babeltrace_thapi -c style log file"